### PR TITLE
<Notification /> - Replace deprecated components in ActionButton

### PR DESCRIPTION
**Pull request by saarkuriel**
### 🔦 Summary
Removed Deprecated `Button` and `TextLink` and replaced them with the new components. 

### ✅ Checklist
- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by mykas** [comment]
Hi, thanks for your PR! There is one unit testing failing as it was tied to TextLink, can you check it?

**Comment by saarkuriel** [comment]
yeah sorry about that, very irresponsible 🤦

**Comment by mykas** [comment]
No probs, thats what CI is for 👌

File: src/Notification/ActionButton/ActionButton.js [hunk 1]
```diff
@@ -11,18 +11,19 @@ const ActionButton = ({ children, onClick, type, link }) => {
...
         {...commonProps}
       >
         {children}
-      </TextLink>
+      </TextButton>
     );
   } else {
     return (
-      <Button height="small" theme="transparent" {...commonProps}>
+      <Button as="button" size="small" skin="transparent" {...commonProps}>
```
Comment (mykas) [inline src/Notification/ActionButton/ActionButton.js]:
```suggestion
      <Button size="small" skin="transparent" {...commonProps}>
```

Comment (mykas) [reply src/Notification/ActionButton/ActionButton.js]: Just a small change.

Comment (saarkuriel) [reply src/Notification/ActionButton/ActionButton.js]: `as="button"` is redundant?

**Pull request merged**